src/main/java/com/celements/course/classes/CourseParticipantClass.java

```diff
@@ -20,10 +21,11 @@
   public static final String CLASS_NAME = "CourseParticipantClass";
   public static final String CLASS_DEF_HINT = CLASS_SPACE + "." + CLASS_NAME;
 
+  public static final List<ParticipantStatus> ignorListForCount = Arrays.asList(

```

Comment:
- illegal naming for constant
- Isn't this CourseService logic?
- I would define them as e.g. DEFAULT_IGNORE_STATES in CourseService